packages/core/src/Stores/client-store.js

```diff
@@ -401,6 +402,16 @@ export default class ClientStore extends BaseStore {
         return typeof client_tnc_status !== 'undefined' && client_tnc_status !== terms_conditions_version;
     }
 
+    @computed
+    get is_social_signup() {
+        return this.account_status?.status?.some(status => status === 'social_signup');
```

Comment:
Any issues with using `includes` instead of `some`?
```suggestion
        return this.account_status?.status?.includes('social_signup');
```
Can call it `has_social_signup` instead of `is_social_signup` since the `account_status` returns returns a string of statuses in an array. 
Regardless of `some` or `includes`, we're checking whether the array contains the string that we're looking for.

Reply:
Will update to `includes`